indexer/src/main/java/au/org/aodn/esindexer/model/DataRecord.java

```diff
@@ -0,0 +1,48 @@
+package au.org.aodn.esindexer.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+// TODO: if more fields are needed to be filtered, please add more columns here
+@Getter
+@Setter
+public class DataRecord{
```

Comment:
I think they all need to reconsider a clearer and consistent name, to be less confusing,

As is growing, the terms 
: data record, metadata record, data, dataset etc. 

What are they? Are they the same thing or different?